src/elements/content-sidebar/activity-feed/activity-feed/ActiveState.js

```diff
@@ -164,25 +169,28 @@ const ActiveState = ({
                         return (
                             <ActivityItem
                                 key={item.type + item.id}
-                                className="bcs-activity-feed-annotation-activity"
-                                data-testid="annotation-activity"
+                                className={!hasReplies && 'bcs-activity-feed-annotation-activity'}
```

Comment:
Can we remove this logic? The class seems unrelated since it's on a parent of the ActivityThread.